lib/libwebsockets.c

```diff
@@ -308,6 +308,9 @@ lws_close_free_wsi(struct lws *wsi, enum lws_close_status reason)
 		if (n)
 			lwsl_debug("closing: shutdown ret %d\n", LWS_ERRNO);
 
+// This causes problems with disconnection when the events are half closing connection
+// FD_READ | FD_CLOSE (33)
+#ifndef _WIN32_WCE
```

Comment:
No you are right, POSIX means that it's a platform that follows posix semantics for its apis, windows basically does that.

On the other hand mbed3 is a case where the operating system itself is a single global event loop.  It has some posix-sounding apis but they are fundamentally different.